conans/test/utils/build_helpers.py

```diff
@@ -0,0 +1,10 @@
+
+import subprocess
+from conans.util.files import decode_text
+from semver import SemVer
+
+
+def get_cmake_version():
```

Comment:
Move to CMake() as static `get_version()` method. I would prefer to return a `Version` object (from conans model), it is able to compare with strings.

Reply:
@memsharded, do you agree?

Reply:
Yes, ``CMake.get_version()`` returning a Version makes sense to me